requirements.txt

```diff
@@ -5,6 +5,8 @@ evelink
 dnspython
 passlib
 requests>=2.9.1
+requests_cache
+json
```

Comment:
I'm able to import json without installing through pip - maybe it comes bundled in a python package installed for dependencies?